netlify.toml

```diff
@@ -1,7 +1,6 @@
 [template]
   incoming-hooks = ["Sanity"]
 
-[template.environment]
-  NEXT_PUBLIC_SANITY_PROJECT_ID="Your Sanity Project Id"
-  NEXT_PUBLIC_SANITY_DATASET="Your Sanity Dataset"
-  
\ No newline at end of file
+  [template.environment]
```

Comment:
could you revert this file change since yaml uses spacing to parse the structure? all else looks good

Reply:
Either ESLint or Prettier is auto-correcting me into the indentation on save, which, for TOML files at least, is valid and parseable.

See the TOML repo: https://github.com/toml-lang/toml
![image](https://github.com/rave-cafe/rave-cafe/assets/4810002/a7a7d2d0-3c62-44b4-8299-67ab46f28188)

It looks like the dot notation triggers an extra indent in our formatter, so [template.environment] is nested (and indented) beneath [template]

I don't feel strongly either way (since both appear valid), but I'm pretty sure the enforcement is coming from our linter settings, so lets either keep it as-is in this PR, or I can figure out how to update our linter settings as part of this PR if you feel strongly.

lmk @daltn

Reply:
i'm good with how it is if it reads correctly by the machinez